socorro/external/crashstorage_base.py

```diff
@@ -630,26 +589,33 @@ def save_raw_and_processed(self, raw_crash, dump, processed_crash,
         # processed crash which is a SocorroDotDict into a pure python
         # dict which we can more easily copy.deepcopy() operate on.
         processed_crash_as_dict = socorrodotdict_to_dict(processed_crash)
+        raw_crash_as_dict = socorrodotdict_to_dict(raw_crash)
 
         for a_store in self.stores.itervalues():
             self.quit_check()
             try:
                 actual_store = getattr(a_store, 'wrapped_object', a_store)
 
-                if hasattr(actual_store, 'is_mutator') and actual_store.is_mutator():
+                if (
+                    hasattr(actual_store, 'is_mutator') and
+                    actual_store.is_mutator()
+                ):
```

Comment:
I don't understand this change. Did anything change here or is it just getting around an 80-char limit?

If it's the latter, I'd rather we didn't change this since it doesn't make the code easier to read.